src/content/docs/apm/agents/java-agent/api-guides/ai-monitoring.mdx

```diff
@@ -37,20 +39,13 @@ If you've disabled the agent with `ai_monitoring.record_content.enabled=false`,
         NewRelic.getAgent().getAiMonitoring.setLlmTokenCountCallback(myTokenCountCallback);
     ```
 
-To use the callback, implement `LlmTokenCountCallback` so that:
-
-* It returns an integer that represents the number of tokens for a particular prompt, completion message, or embedding. 
-* If the values are less than or equal to 0, `LlmTokenCountCallbacks` will not be attached to an event.
-
-Keep in mind that this API should be called only once. Calling this API multiple times will replace each previous callback. 
+To use the callback, implement `LlmTokenCountCallback` so that it returns an integer that represents the number of tokens for a particular prompt, completion message, or embedding. If the values are less than or equal to `0`, `LlmTokenCountCallbacks` will not be attached to an event. Keep in mind that this API should be called only once. Calling this API multiple times will replace each previous callback. 
```

Comment:
```suggestion
To use the callback, implement `LlmTokenCountCallback` so that it returns an integer that represents the number of tokens for a particular prompt, completion message, or embedding. If the values are less than or equal to `0`, `LlmTokenCountCallbacks` won't attach to an event. Keep in mind that you should only call this API once. Calling this API multiple times will replace each previous callback. 
```